GothicLlamasAPI/src/main/java/com/kainos/ea/objects/JobRoleAndCapabilityMapper.java

```diff
@@ -0,0 +1,19 @@
+package com.kainos.ea.objects;
+
+import org.jdbi.v3.core.mapper.RowMapper;
+import org.jdbi.v3.core.statement.StatementContext;
+
+import java.sql.ResultSet;
+import java.sql.SQLException;
+
+public class JobRoleAndCapabilityMapper implements RowMapper<CapabilitiesAndRoles>{
+    @Override
+    public CapabilitiesAndRoles  map(ResultSet r, StatementContext ctx) throws SQLException
+    {
```

Comment:
Couple of minor formatting issues - a linter would be helpful to capture this stuff automatically!
```suggestion
public class JobRoleAndCapabilityMapper implements RowMapper<CapabilitiesAndRoles> {
    @Override
    public CapabilitiesAndRoles map(ResultSet r, StatementContext ctx) throws SQLException {
```